prebuilts/rabbitmq.toml

```diff
@@ -0,0 +1,71 @@
+#:schema ./schema.json
+# 待审查
+
+id = "rabbitmq"
+name = "RabbitMQ"
+
+# TODO Download svg icon from https://www.svgrepo.com/svg/303576/rabbitmq-logo
+icon = "https://raw.githubusercontent.com/zeabur/service-icons/main/marketplace/rabbitmq.svg"
+
+description = "RabbitMQ is a widely used open-source message broker that implements the Advanced Message Queuing Protocol (AMQP)"
+
+[source]
+image = "rabbitmq:3-management"
+
+[[ports]]
+id = "database"
+port = 5672
+type = "TCP"
+
+[[volumes]]
+id = "data"
+# mountPath = "/var/lib/rabbitmq"
+dir = "/data/rabbitmq" # TODO
```

Comment:
Not sure what you want to do. You should fill `/var/lib/rabbitmq` if RabbitMQ saves persistent files there.

Reply:
https://www.rabbitmq.com/backup.html